src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDistribucion.kt

```diff
@@ -0,0 +1,30 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDistribucion(val ciudad:Ciudad){
+
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor:Vendedor){
+        if(!vendedores.contains(vendedor)){
+            vendedores.add(vendedor)
+        }else{
+            throw Exception("No se puede agregar dos veces el mismo vendedor")
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
        if(vendedores.contains(vendedor)){
            throw Exception("No se puede agregar dos veces el mismo vendedor")
        }
        
        vendedores.add(vendedor)
```